java/code/src/com/suse/manager/webui/services/impl/SaltService.java

```diff
@@ -122,14 +123,16 @@
 /**
  * Singleton class acting as a service layer for accessing the salt API.
  */
-public class SaltService implements SystemQuery {
+public class SaltService implements SystemQuery, SaltApi {
 
     private final Batch defaultBatch;
 
     /**
      * Singleton instance of this class
      */
-    public static final SystemQuery INSTANCE = new SaltService();
+    private static final SaltService INSTANCE_SALT_SERVICE = new SaltService();
+    public static final SystemQuery INSTANCE = INSTANCE_SALT_SERVICE;
+    public static final SaltApi INSTANCE_SALT_API = INSTANCE_SALT_SERVICE;
```

Comment:
This is confusing, why they both are pointing to same? Are they even being used?

Reply:
The reason is to have the variable be declared with the interface type rather then `SaltService` to there is no accidental mixup. It also helps to track where each interface is used right now. 
In the end those static instances are just temporary until further refactorings.